libs/dh/metering-point/feature-overview/src/lib/breadcrumbs/dh-breadcrumbs.module.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * @license
+ * Copyright 2021 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { RouterModule } from '@angular/router';
+import { TranslocoModule } from '@ngneat/transloco';
+
+import { WattIconModule } from '@energinet-datahub/watt';
+
+import { DhBreadcrumbsComponent } from './dh-breadcrumbs.component';
+
+@NgModule({
+  declarations: [DhBreadcrumbsComponent],
+  exports: [DhBreadcrumbsComponent],
+  imports: [CommonModule, RouterModule, TranslocoModule, WattIconModule],
+})
+export class DhBreadcrumbsScam {}
```

Comment:
question: I'm not sure if we agreed to use inline SCAMs or to have them separately in `.module.ts` files. Do you remember?

Reply:
By now I'm confused as well. My understanding is that we're using inline SCAMs when the component and its HTML/SCSS consist of only a few lines but if they grow bigger, then they are moved to separate files.